### ISSUE TITLE
Informations in the DatePicker are half-translated

### ISSUE THREAD

**Issue by Xaalek** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2318]
Hello,
As you can see on the screenshot below, the text in the datepicker is half-translated :
![image](https://user-images.githubusercontent.com/54487782/117260601-8ad81b00-ae4f-11eb-8e8c-0b20feb13e7b.png)
Blue area is translated. Red areas are not translated.

Here is the calendar of my computer in comparison :
![image](https://user-images.githubusercontent.com/54487782/117260354-49477000-ae4f-11eb-9ccd-50c455c608dd.png)

A mix of multiple languages in the same panel is not great for the user experience. 
I think we should consider writing everything in English or translating everything into the language of the computer.

**Comment by Keboo** [comment]
Agreed, this looks like a bug. We should be respecting the language that is specified in the current culture.

**Comment by woltek** [comment]
Got the same thing,

Seems like Language.GetSpecificCulture() is not returning Current culture
use CultureInfo.CurrentCulture instead.

![image](https://user-images.githubusercontent.com/9975500/158605040-bac43be4-f6d6-4189-b3d5-fb2664e52e18.png)

**Comment by Xaalek** [comment]
@woltek wich file ?

**Comment by Xaalek** [comment]
@woltek If we change this, we won't be able to manually define a culture on the DatePicker.

I find a way to change the default culture by the current one.

We have to add this in the App.OnStartup() event
```
Thread.CurrentThread.CurrentCulture = CultureInfo.CurrentCulture;
            Thread.CurrentThread.CurrentUICulture = CultureInfo.CurrentCulture;
            FrameworkElement.LanguageProperty.OverrideMetadata(typeof(FrameworkElement), new FrameworkPropertyMetadata(
                        XmlLanguage.GetLanguage(CultureInfo.CurrentCulture.IetfLanguageTag)));
```

Moreover, it's commented in the demo app : https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/MainDemo.Wpf/App.xaml.cs#L20

**Comment by Keboo** [comment]
I agree that the method documented in the demo app is what should be done to set the language of your app. This is because WPF does not follow the system language. See https://github.com/dotnet/wpf/issues/1946

However, we have had bugs where it code was not respecting the current culture. As long as the calendar properly updates after setting the language, I would say this is not an issue.

**Comment by Xaalek** [comment]
Before saying that it is not a bug, we must test if when we put a different language of the computer, the information in blue is well translated in this language. I forgot to test this

**Comment by Xaalek** [comment]
@Keboo 
I tried to put the application in English with `CultureInfo("en-US")` while my windows is in French. The DatePickers are well translated into English. 
I also tried to use CultureInfo.CurrentCulture and the DatePickers are well translated into French.

So we can say that the problem is solved. We just need to write this information in the wiki so that other people know how to configure the language of an application that use MaterialDesignInXaml. It will also allow people to not think it's a bug whereas they just forgot a few lines of code.
